### PR TITLE
Some more translations into Hindi

### PR DESCRIPTION
**Pull request by Rishabhbhat**
I made some more translations into Hindi.

### CODE REVIEW

**Comment by Rishabhbhat** [comment]
Use the second one(Update duckduckgo.po)

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -83,15 +83,15 @@ msgctxt "noresults"
 msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
-msgstr ""
+msgstr "अगर आपको लगता है कि इस खोज के लिए परिणाम होने चाहिए, तो %sइधर दबाएँ%s"
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo!%s"
-msgstr ""
+msgstr "%sडकडकगो फैलाने में मदद करें!%s"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `Duckduckgo` name will not be translated as it is a Trademark. Please leave it as it is.    
Do not do `डकडकगो`.

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -83,15 +83,15 @@ msgctxt "noresults"
...
 
 msgctxt "frontpage"
 msgid "%sHelp Spread DuckDuckGo%s."
-msgstr ""
+msgstr "%sडकडकगो फैलाने में मदद करें!%s"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `Duckduckgo` term will not be translated as it is a Trademark. Please leave it as it is.

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -123,54 +123,54 @@ msgstr ""
...
 #. It is not "About DuckDuckGo".
 msgid "About"
-msgstr "के बारे में"
+msgstr "इसके बारे में"
 
 #. This is the side bar "About" link
 msgctxt "static_page"
 msgid "About"
-msgstr ""
+msgstr "डकडकगो के बारे में"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `Duckduckgo` term will not be translated as it is a Trademark. Please leave it as it is.

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -123,54 +123,54 @@ msgstr ""
...
 
 msgctxt "feedback form"
 msgid "Ad is annoying"
-msgstr ""
+msgstr "विज्ञापन परेशान कर रहा है"
 
 msgctxt "feedback form"
 msgid "Ad is inappropriate"
-msgstr ""
+msgstr "विज्ञापन अनुचित है"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `अप्रयोज्य` or `अनुपयुक्त` will be the correct term here.

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -123,54 +123,54 @@ msgstr ""
...
 
 msgctxt "feedback form"
 msgid "Ad is irrelevant"
-msgstr ""
+msgstr "विज्ञापन बेजोड़ है"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `बेजोड़` does not look good. `अप्रासंगिक` sounds better.

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -123,54 +123,54 @@ msgstr ""
...
-msgstr ""
+msgstr "विज्ञापन सन्देहजनक है"
 
 #. From Home page, options to "Add DuckDuckGo" to browser or set as home page.
 msgid "Add DuckDuckGo"
 msgstr "डकडकगो  जोड़ें  "
 
 msgid "Add DuckDuckGo as a search engine"
-msgstr ""
+msgstr "डकडकगो को खोज यन्त्र के रूप में जोडें"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `डकडकगो` ---> `DuckDuckGo`

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 7]
```diff
@@ -123,54 +123,54 @@ msgstr ""
...
 
 msgid "Add DuckDuckGo to %s"
-msgstr ""
+msgstr "डकडकगो को  %s में जोडें"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `डकडकगो` ---> `DuckDuckGo`

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 8]
```diff
@@ -285,11 +285,10 @@ msgstr "उत्तर"
 
 #. In feedback page, first box. https://duckduckgo.com/feedback 
 msgid "Answers to most DuckDuckGo questions, like:"
-msgstr "Duckduckgo adhikamsh prashnon ke uttar, jaise:"
-
+msgstr "डकडकगो के ज़्यादातर सवालों के जवाब, जैसे:"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `डकडकगो` ---> `DuckDuckGo`

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 9]
```diff
@@ -334,10 +333,10 @@ msgstr ""
 
 msgctxt "homepage ATB modal"
 msgid "At DuckDuckGo, we agree."
-msgstr ""
+msgstr "डकडकगो में, हम इस बात से सहमत हैं"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: `डकडकगो` --> `DuckDuckGo`

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 10]
```diff
@@ -645,7 +644,7 @@ msgid "Chat"
 msgstr "बातचीत "
 
 msgid "Check %sMake this the current search engine%s"
-msgstr "%s %s"
+msgstr "%sइसे अपना खोज यन्त्र बनाएँ%s"
```
Comment (soumendrak) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: Check translation missing. This is in the context to tick mark on `Make this the current search engine`.

**Comment by Rishabhbhat** [comment]
@soumendrak Thanks. However, अनुचित seems correct to me. अप्रयोज्य or अनुपयुक्त will mean irrelevant.

**Comment by jbarrett** [comment]
Many thanks @Rishabhbhat and @soumendrak! Is this ready to go?

**Comment by soumendrak** [comment]
Thanks @Rishabhbhat for your contributions.

**Comment by Rishabhbhat** [comment]
Okay, can be done now.

**Comment by jbarrett** [comment]
:+1: Many thanks!

**Comment by Rishabhbhat** [comment]
😊

**Pull request merged**